laia/data/transforms/vision/transforms.py

```diff
@@ -33,4 +33,60 @@ def __repr__(self):
         return format_string
 
 
+class ToPreparedTensor(object):
```

Comment:
This name is confusing. Can you think of a better name?

Reply:
Maybe ToImageTensor?